spellbook/tests/sudoswap/ethereum/sudoswap_ethereum_assert_swaps.sql

```diff
@@ -0,0 +1,90 @@
+-- Bootstrapped correctness test against legacy Postgres values.
+-- Also manually check etherscan info for the first 5 rows
+WITH 
+    raw_swaps as (
+        WITH
+            pairs_created as (
+                SELECT 
+                    _nft as nftcontractaddress
+                    , _initialNFTIDs as nft_ids
+                    , _fee as initialfee
+                    , _assetRecipient as asset_recip
+                    , output_pair as pair_address
+                    , call_block_time as block_time
+                FROM {{ source('sudo_amm_ethereum','LSSVMPairFactory_call_createPairETH') }}
+                WHERE call_success
+                AND _nft = '0xef1a89cbfabe59397ffda11fc5df293e9bc5db90'
+            ),
+
+            fil_swaps as (
+                SELECT 
+                    * 
+                FROM (
+                    SELECT 
+                        contract_address
+                        , call_tx_hash
+                        , call_trace_address
+                        , call_block_time
+                        , call_block_number
+                        , call_success
+                        , tokenRecipient as call_from
+                        , 'Sell' as trade_category 
+                    FROM sudo_amm_ethereum.LSSVMPair_general_call_swapNFTsForToken
```

Comment:
any reason your recent commit switched back from {{ source( ) }} syntax to hardcode? looks like line 14 still has it, i think you're good to switch back

Reply:
i can continue to dig deeper into the models for now, tests will come last, but i'll look closer at this test soon

Reply:
Ah I think I was copying and pasting from compiled to dune and back to make sure the query ran correctly. I'll correct it tmr